extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/extractors/SoundcloudStreamExtractor.java

```diff
@@ -121,7 +121,8 @@ public long getLength() {
 
     @Override
     public long getTimeStamp() throws ParsingException {
-        return getTimestampSeconds("(#t=\\d{0,3}h?\\d{0,3}m?\\d{1,3}s?)");
+        final var timestamp = getTimestampSeconds("(#t=\\d{0,3}h?\\d{0,3}m?\\d{1,3}s?)");
+        return timestamp == -2 ? 0 : timestamp;
```

Comment:
For a future PR, you could make `getTimestampSeconds()` return an `Optional<>`. Also, all regex-related functions should ideally only take `Pattern`s, so we can avoid rebuilding regexes every time.